src/components/client.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Represents the client
+ * @type {module:eris.Client}
+ */
+const Eris = require('eris')
+const { Collection } = require('eris')
+module.exports = new Eris(process.env.BOT_TOKEN, {
+  messageLimit: 10
+})
+
+module.exports.wildbeastVoiceConnections = new Collection(require('../classes/VoiceConnection'))
```

Comment:
I'm a bit iffy on this variable naming, could we name it something like `voiceConnectionManager`?